README.md

```diff
@@ -61,16 +61,6 @@
 
```

Comment:
![image](https://github.com/Peter-Kibet/Math_Magician_Calculator/assets/93039359/23095534-9fe2-4c5b-86b1-0dfb7499649e)

**[OPTIONAL]**

- [ ] It would be nice if you remove the placeholder texts from your readme file to make your readme clean and professional 👍